resources/views/lists/tools.blade.php

```diff
@@ -15,100 +15,46 @@
                         <label for="spotify_name">name your Spotify playlist</label>
                         <input type="text" class="form-control" id="spotify_name" placeholder="Enter name">
                         <p class="mt-3">This upload will create a new playlist within your Spotify account using the song titles. It will not share photos and other information from this app. </p>
-                        <a class="btn" href="#}">Upload</a>
+                        <a class="btn btn_save" href="#}">Upload</a>
```

Comment:
`href="#}"` this looks trage, but i'm guessing you will change it later on anyways